.github/workflows/docker-build.yaml

```diff
@@ -18,5 +18,5 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
           registry: docker.pkg.github.com
-          repository: epiforecasts/covid.german.forecasts/covidgermanforecasts
+          repository: epiforecasts/europe-covid-forecast/europecovidforecast
```

Comment:
what is this repository exactly?

Reply:
this is the name of the docker container in this repo.